src/index.html

```diff
@@ -5,8 +5,365 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          class="logo_image"
+          alt="Moyo Logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__block">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__block">
```

Comment:
```suggestion
          </li>
          
          <li class="nav__block">
```

Reply:
<img width="1098" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/a24c27e6-01d1-4aae-97cc-2d807303b888">
Fix this in all file